### DISCUSSION TITLE
请教 `#scaler-resizes-only=no`参数的进一步解释

### DISCUSSION THREAD

**Discussion by linwhth** [dyphire/mpv-config#81] [Q&A] [answered]
`#scaler-resizes-only=no               # 默认 yes，当未进行缩放处理时，使用 bilinear 算法完美还原，禁用时，即使未改变大小，也使用指定的 scale 值进行处理，可以修正 nnedi3 和 ravu 引入的半像素偏移（但没必要）`

我想请教一下为什么修正 nnedi3 和 ravu 引入的半像素偏移是不必要的呢？
是因为半像素偏移对图像影响不大吗？

**Comment by dyphire** [comment, marked as answer]
nnedi3 和 ravu 引入的半像素偏移可以抑制图像的原始伪影，消除它反而减弱了着色器的效果

不必要的原因是字面意思，什么火眼金睛不对比能看出半个像素的偏移？

**Reply by linwhth** [reply]
多谢解答！
